ssl/statem/extensions_clnt.c

```diff
@@ -108,8 +108,8 @@ static int use_ecc(SSL *s)
     return i < end;
 }
 
-int tls_construct_ctos_ec_pt_formats(SSL *s, WPACKET *pkt, X509 *x,
-                                     size_t chainidx, int *al)
+int tls_construct_ctos_ec_pt_formats(SSL *s, WPACKET *pkt, unsigned int context,
```

Comment:
It's not hiding, no more than SSL hides things or WPACKET does.  I think it's better and more maintainable, and gave the best reasoning that I could.  Oh well.